cypress/e2e/CMS/$InfographicCreation/index.js

```diff
@@ -0,0 +1,298 @@
+/// <reference types="Cypress" />
+import { And, When } from 'cypress-cucumber-preprocessor/steps';
+import { extractImgName } from "../../../utils/extractImgName.js";
+
+
+const siteSection = Cypress.env('test_site_section');
+const randomStr = Cypress.env('randomStr');
+const frontEndBaseUrl = Cypress.env('front_end_base_url');
+
+And('user fills out the following fields', (dataTable) => {
+    for (let { fieldLabel, value, field_name } of dataTable.hashes()) {
+        if (fieldLabel === 'Pretty URL') {
+            value = `${value}-${randomStr}`;
+        }
+        cy.get(`input[name^='${field_name}']`).as('inputField').parent().find('label').should('include.text', fieldLabel);
+        cy.get('@inputField').type(value);
+    }
+});
+
+And('user types {string} into Caption text field', (value) => {
+    cy.getNthIframe("iframe[class='cke_wysiwyg_frame cke_reset']", 1).find('p').type(value);
+});
+
+And('user uploads test file {string}', (fileName) => {
+    cy.fixture(fileName, { encoding: null }).as('fixture')
+    cy.get('input[type="file"]').first().selectFile('@fixture');
+    cy.wait(2000);
```

Comment:
same - use wait step in a feature file instead

Reply:
removed